docs/resources/migration.md

```diff
@@ -0,0 +1,79 @@
+
+What this means is that a first step for migrating over your components is setting up the development Environment - either by simply selecting the relevant [base Environment](/building-with-bit/environments#pre-built-environments) that Bit supplies (react, node, react-native for now - Angular, Vue etc coming soon!), or by [extending these base Environments](building-with-bit/environments#customizing-environments) by customizing configurations, e.g. by adding your own tsconfig.
+
+
+### Porting over components
+
+This stage should mainly be a simple copy-paste job - see [here](/building-with-bit/pre-existing-components#add-pre-existing-components) for details of adding existing components and basic requirements that Harmony has introduce for component structure.
+
+### Configure the workspace
+
+Our [workspace documentation](/building-with-bit/workspace) details exactly how to configure your `workspace.jsonc` file to set environments, dependency configurations and more as required.
+
+
+Once you have the components tracked in your Bit workspace and have configured the relevant environment for them in the workspace's variants section, you should now be able to compile, tag and export them much as you would have with previous versions of Bit - though it's not all 100% the same, so it's worth going through the Harmony intro docs to make sure you're familiar with what's changed (and what hasn't). 
+And now you have a whole raft of new features at your disposal - not least the local development UI, try it by running `bit start`!
+
+:::note Harmony components can only be exported to Harmony scopes
+Make sure to create your Harmony scope/s on bit.dev as required and map your components to the relevant remote scope via the workspace configuration **before** exporting your ported components
+:::
+
+
+## New Features in Harmony
+
+Why go through all the trouble? you may ask yourself. Well, here's just a sample of the added features that Harmony provides, and the list is ever growing.
+
+One of the main leaps forward we've made with Harmony is that it is extremely extendible - both by ourselves and by users themselves - so Harmony's feature set is constantly expanding.
+That said, these are the major new features that have been introduced with Harmony:
+- **Component Development Environments** Shareable components which contain the configurations for the full component lifecycle, to both reduce setup time and to introduce component development standardization across the organization  
+- **Documentation** Use `.docs.` files to document your components at source
+- **Compositions** Use `.compositions.` files to demonstrate and illustrate your component with its variants and inside complex application contexts, all rendered as part of your component's documentation
+- **Ripple CI** Propagating CI along the dependency tree, so you know how changes upstream will affect dependencies, even before they adopt the changes
```

Comment:
Really really dont assume that people who have gotten here understand all the benefits and are fully bent on migration. This might be where they give up unless we give them reasons to persist with the migration